src/entities/models/dto.py

```diff
@@ -53,3 +53,9 @@ class FilingPeriodDTO(BaseModel):
     end_period: datetime
     due: datetime
     filing_type: FilingType
+
+
+class UpdateValueDTO(BaseModel):
+    model_config = ConfigDict(from_attribute=True)
+
+    value: Any
```

Comment:
let's have the type be more strict, I think we can accept the simple types for now; something like contact_info will need its own model. I'm thinking string, int, float, and bool for now.

Reply:
Yup, sounds good.  Updated, added a pytest for it, too.  Also merged main and resolved some conflicts so it's the first commit after this review.